config/environment.js

```diff
@@ -6,6 +6,22 @@ module.exports = function (environment) {
     environment,
     rootURL: '/',
     locationType: 'history',
+    announce: {
+      maintenance: {
+        enabled: '{{ANNOUNCE_MAINTENANCE_ENABLED}}',
+        message: '{{ANNOUNCE_MAINTENANCE_MESSAGE}}',
+      },
+      newDeployment: {
+        enabled: '{{ANNOUNCE_NEW_DEPLOYMENT_ENABLED}}',
+        message: '{{ANNOUNCE_NEW_DEPLOYMENT_MESSAGE}}',
+      },
+      testing: {
+        enabled: '{{ANNOUNCE_TESTING_ENABLED}}',
+        message: '{{ANNOUNCE_TESTING_MESSAGE}}',
+      },
+    },
+    appName: 'Lokaal Mandatenbeheer',
+    environmentName: '{{ENVIRONMENT_NAME}}',
```

Comment:
could you create a pr that sets this in the app?

Reply:
Pr for setting the environment name in the LMB app https://github.com/lblod/app-lokaal-mandatenbeheer/pull/215